doc/src/build-instructions.md

```diff
@@ -272,17 +280,85 @@ $ idf.py build
 
 This command, once completed, will create the Espressif bootloader, partition table, and AtomVM binary.  The last line of the output should read something like the following:
 
-    Project build complete. To flash, run this command:
-    ~/.espressif/python_env/idf5.1_py3.11_env/bin/python ~/esp/esp-idf-v5.1/components
-    /esptool_py/esptool/esptool.py -p (PORT) -b 921600 --before default_reset
-    --after hard_reset --chip esp32 write_flash --flash_mode dio --flash_size detect
-    --flash_freq 40m 0x1000 build/bootloader/bootloader.bin 0x8000
-    build/partition_table/partition-table.bin 0x10000 build/atomvm-esp32.bin
-    or run 'idf.py -p (PORT) flash'
+```
+Project build complete. To flash, run:
+ idf.py flash
+or
+ idf.py -p PORT flash
+or
+ python -m esptool --chip esp32 -b 921600 --before default_reset --after hard_reset write_flash --flash_mode dio --flash_size 4MB --flash_freq 40m 0x1000 build/bootloader/bootloader.bin 0x8000 build/partition_table/partition-table.bin 0x10000 build/atomvm-esp32.bin 0x1d0000 ../../../build/libs/esp32boot/esp32boot.avm
+or from the "/home/winford/work/AtomVM/src/platforms/esp32/build" directory
```

Comment:
from the "/home/winford/work/AtomVM/src/platforms/esp32/build" directory -> from your "AtomVM/src/platforms/esp32/build" directory

or similar..

Reply:
Thank you!!! That was a last minute addition to the docs (wanted to get the library output in there)… I forgot it had full path info included.

Reply:
I added some clarification about the path to use, and changed this example output to a less specific path and used the user name "joe", since we started using that as a placeholder for the user name in other documentation examples where a username is expected in program output.